CORPSE_integrate.py

```diff
@@ -20,32 +32,39 @@ def fsolve_wrapper(SOM_list,T,theta,Ndemand,Ctransfer,inputs,clay,params):
         for n in range(len(fields)):
             SOM_dict[fields[n]]=asarray(SOM_list[n])
 
+    if runtype == 'Final':
+        Nlitter = Ndemand * timecoeff(times)
+    else:
+        Nlitter = Ndemand
+
     # Call the CORPSE model function that returns the derivative (with time) of each pool
-    deriv=CORPSE_deriv.CORPSE_deriv(SOM_dict,T,theta,Ndemand,Ctransfer,params,claymod=CORPSE_deriv.prot_clay(clay)/CORPSE_deriv.prot_clay(20))
+    deriv=CORPSE_deriv.CORPSE_deriv(SOM_dict,T,theta,Nlitter,Ndemand,params,claymod=CORPSE_deriv.prot_clay(clay)/CORPSE_deriv.prot_clay(20))
 
     for pool in inputs.keys():
-        deriv[pool]+=inputs[pool]
+        if runtype == 'Final':
+            deriv[pool]+=inputs[pool] * timecoeff(times)
```

Comment:
This means that plant allocation to mycorrhizae (intermediate pools) will follow the same annual pattern as leaf litter since both are included in the inputs. Is that what you meant to do? I would expect mycorrhizal allocation to have a different seasonal pattern.